src/kanata.rs

```diff
@@ -312,12 +321,12 @@ impl Kanata {
                                 Err(_) => {
```

Comment:
Any clients that have failed the keep-alive should result in an error being thrown here the next time the server tries to write to them and then get cleaned up below